docs/components/accordion/README.md

```diff
@@ -85,28 +117,387 @@
+    <cdr-accordion-item
+      id="default-1"
+      label="How do I find my member number?">
+      <cdr-text tag="p">
+          Find your member number online. You can also call 
+          Customer Support at 1-800-426-4840 (U.S. and Canada) or 1-253-891-2500 (International).
+      </cdr-text>
+    </cdr-accordion-item>
+    <cdr-accordion-item
+      id="default-2"
+      label="Does every member get an Annual Dividend?">
+      <cdr-text tag="p">
+          Only active REI Co-op members receive an Annual Dividend notice. To be an active 
+          member, you need to make net merchandise or shipping purchases (purchases minus credits and returns) 
+          of at least $10 per year, unless you joined during that calendar year.
+      </cdr-text>
+    </cdr-accordion-item>
+    <cdr-accordion-item
+      id="default-3"
+      label="When does my dividend expire?">
+      <cdr-text tag="p">
+          Your dividend expires on Jan. 3, just under two years after it has been issued. 
+          or example, your 2018 dividend earned on 2017 purchases will expire in January 2020.
+      </cdr-text>
+    </cdr-accordion-item>
+  </cdr-accordion>
+```
+
+</cdr-doc-example-code-pair>
+
+## Compact
+
+Reduced spacing around title and content body. Also, smaller font sizes resulting in an overall denser display of content.
+
+<cdr-doc-example-code-pair repository-href="https://github.com/rei/rei-cedar/tree/18.07.1/src/components/accordion" sandbox-href="https://codesandbox.io/s/m9jm5rw1zx">
+
+```html
+  <cdr-accordion :compact="true">
+    <cdr-accordion-item
+      id="compact-1"
+      label="Why buy used gear?"
+    >
+      <cdr-text tag="p">
+        Used Gear Beta is one way we are experimenting expanding opportunities 
+        to enjoy life outdoors and bringing value to our members.
+      </cdr-text>
+    </cdr-accordion-item>
+    <cdr-accordion-item
+      id="compact-2"
+      label="What's your cancellation policy?"
+    >
+      <cdr-text tag="p">
+        Orders may be cancelled within 30 minutes of placing your order online. 
+        After 30 minutes, your order will begin processing through our fulfillment center and cannot be cancelled.
+      </cdr-text>
+    </cdr-accordion-item>
+    <cdr-accordion-item
+      id="compact-3"
+      label="When will my order arrive?"
+    >
+      <cdr-text tag="p">
+        REI Co-op Used Gear Beta orders can take up to 3-4 business days to ship out. When your order ships, 
+        we'll send you a shipping confirmation email that contains your tracking information. Shipping time is generally 3-5 business days.
+      </cdr-text>
+    </cdr-accordion-item>
+  </cdr-accordion>
+```
+
+</cdr-doc-example-code-pair>
+
+## Border Aligned
+
+Border aligns to the title text and expand/collapse icon.
+
+<cdr-doc-example-code-pair :background-toggle="false" repository-href="https://github.com/rei/rei-cedar/tree/18.07.1/src/components/accordion" sandbox-href="https://codesandbox.io/s/m9jm5rw1zx">
```

Comment:
I think this is 18.07.2 now

Reply:
thats fine this will move to 18.07.3 before this goes live as part of release

Reply:
updated to .2 anyway